css/style.css

```diff
@@ -24,14 +24,20 @@
 
 .custom-focus:focus {
     outline: none; 
-    border:1px solid #e04747; 
-    box-shadow: 0 0 10px #d14b4b;
-}
+    border:1px solid #666666; 
+    box-shadow: 0 0 10px #666666;
+} 
 
 .btn {
-    color: #666666;
+    background-color: #666666;
+}
+
+#navbardrop {
+    outline: none; 
+    border: none; 
+    box-shadow: none;
 }
 
-h5 {
+.text-header {
```

Comment:
I'm not sure if this is a **Bootstrap 4's class**. If not, maybe you should change to `header-text` & move it up to `header` CSS declarations closer to `header-tittle-brand` classes?